intel_extension_for_transformers/llm/library/jblas/jblas/jit_blas_weight_compression.h

```diff
@@ -1292,13 +1378,13 @@ using GemmSKernelDynamicS4FullRangeFp32KBlock = jblas::wrapper::gemm_kblock::Gem
         jblas::epilogue::gemm::AccumulatorWriteBackFp32>,
     jblas::utils::parallel::Parallel2DGemmKBlockFixed>;
 
-using GemmDynamicS4Fp32PerChannelN = jblas::wrapper::gemm_pack_weight::GemmInterfaceAB<
+using GemmDynamicS8Fp32PerChannelN = jblas::wrapper::gemm_pack_weight::GemmInterfaceParallelAB<
     jblas::wrapper::gemm_pack_weight::GemmLauncherPackWeight<
-        DefaultISA, jblas::gemm::GemmCore_Row_NN_16x48_AMX_INT8_ss,
+        DefaultISA, jblas::gemm::GemmCore_Row_NN_16x48_AMX_S8S8,  //
```

Comment:
Is this gemmcore's coretype==AMX_INT8_16x48_SS?

Reply:
AMX_INT8_16x48 is okay. For now, only NTILE matters. After the support of asymmetric weight, we need to split it into two types.